packages/core/src/extension/create-extension-web-worker.ts

```diff
@@ -0,0 +1,29 @@
+export function createExtensionWebWorker(url: string): Worker {
+    let worker: Worker;
```

Comment:
It looks like we don't need this `let`, the function can be returned at L20 directly.